src/wifes_calib.py

```diff
@@ -793,36 +793,55 @@ def derive_wifes_telluric(cube_fn_list,
             ex_data = numpy.loadtxt(extract_in_list[i])
             obs_wave = ex_data[:,0]
             obs_flux = ex_data[:,1]
+        
+        # Filtering nan values in case of missing flux values
+
+        filter_nan = ~numpy.isnan(obs_flux)
+        obs_wave_nan = obs_wave[filter_nan]  
+        obs_flux_nan = obs_flux[filter_nan]
```

Comment:
Do you still get NaNs when running extract_wifes_stdstar even after the new nansum? Or is this just in case? 

If it's just in case, I'd say it's better to throw an error rather than keep running.

Reply:
Yes, I still get NaNs after the new `nansum` in `extract_wifes_stdstar`. But it is because the `nansum` in `extract_wifes_stdstar` is just been used for getting some indices for obtaining `maxind` but not for getting the observing wavelength and the flux (i.e. `lam_array` and `std_flux`), so we still need to take care of those NaNs. What I've done is to take care of the NaNs in `extract_wifes_stdstar` so any further function that uses it, does not raise any issue. Please, let me know if you find this okay.

Reply:
Ah, that's perfect, much more robust, thanks.